float32 value1
float32 value2
---
bool success
string message